services/istio/1.14.1/defaults/cm.yaml

```diff
@@ -25,5 +25,5 @@ data:
         labels:
           prometheus.kommander.d2iq.io/select: "true"
     global:
-      image: "${kubetoolsImageRepository:=bitnami/kubectl}"
-      tag: "${kubetoolsImageTag:=1.23.6}"
+      image: bitnami/kubectl
+      tag: 1.23.6
```

Comment:
we should bump this to be the same everywhere to avoid multiple versions of kubectl, however it's not strictly related to this change so maybe we can do it in a follow-up and see if it breaks istio in any way

Reply:
yeah definitely, I can follow up with that change

Reply:
actually.... our CI doesn't actually deploy Istio, so if we do update it I will have to do some manual testing 😓